lua/presence/init.lua

```diff
@@ -509,8 +509,8 @@ function Presence:get_nvim_socket_paths(on_done)
     if self.os.is_wsl then
         -- TODO: There needs to be a better way of doing this... no support for ss/netstat?
         -- (See https://github.com/microsoft/WSL/issues/2249)
```

Comment:
Also for what it's worth, this if clause might not be necessary anymore.

The TODO mentions `ss` not working but I'm on WSL2 it's included with the default Ubuntu distro and finds the nvim sockets just fine.

```console
❯ uname -r
5.15.133.1-microsoft-standard-WSL2
❯ which ss
/usr/bin/ss
❯ ss -lx | grep nvim
u_str LISTEN 0      32               /home/andrey/.run/nvim.1168028.0 2683110            * 0
u_str LISTEN 0      32               /home/andrey/.run/nvim.1165275.0 2661077            * 0
```